src/Core/Domain/Product/Query/GetProductForEditing.php

```diff
@@ -39,13 +40,22 @@ class GetProductForEditing
     private $productId;
 
     /**
-     * GetEditableProduct constructor.
+     * @var ShopId
+     */
+    private $shopId;
+
+    /**
+     * GetProductForEditing constructor.
      *
      * @param int $productId
+     * @param ShopId $shopId
      */
-    public function __construct(int $productId)
-    {
+    public function __construct(
+        int $productId,
+        ShopId $shopId
+    ) {
         $this->productId = new ProductId($productId);
+        $this->shopId = $shopId;
```

Comment:
what about shop group id? are we planning to handle this use case? or is it not needed?

Reply:
No the group shop context is not handled by the product page (nor the Product object for that matter)